ts/features/walletV3/transaction/screens/WalletTransactionDetailsScreen.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import {
+  Dimensions,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  View
+} from "react-native";
+import { IOColors, IOStyles } from "@pagopa/io-app-design-system";
+import { RouteProp, useRoute } from "@react-navigation/native";
+
+import { WalletTransactionParamsList } from "../navigation/navigator";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import TopScreenComponent from "../../../../components/screens/TopScreenComponent";
+import FocusAwareStatusBar from "../../../../components/ui/FocusAwareStatusBar";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { useOnFirstRender } from "../../../../utils/hooks/useOnFirstRender";
+import { walletTransactionDetailsGet } from "../store/actions";
+import {
+  walletTransactionDetailsPotSelector,
+  walletTransactionDetailsSelector
+} from "../store";
+import { fetchPsp } from "../../../../store/actions/wallet/transactions";
+import { Psp } from "../../../../types/pagopa";
+import WalletTransactionInfoSection from "../components/WalletTransactionInfoSection";
+import { WalletTransactionHeadingSection } from "../components/WalletTransactionHeadingSection";
+
+export type WalletTransactionDetailsScreenParams = {
+  transactionId: number;
+};
+
+export type WalletTransactionDetailsScreenProps = RouteProp<
+  WalletTransactionParamsList,
+  "WALLET_TRANSACTION_DETAILS"
+>;
+
+const styles = StyleSheet.create({
+  iosHeaderScroll: {
+    position: "absolute",
+    height: Dimensions.get("window").height,
+    top: -Dimensions.get("window").height,
```

Comment:
same thing as the loading, imho we could move this to a variable; what do you think?

Reply:
Yep, a way better! Addressed into b1f115e71a062d258ab1ddeea1edb9eb100c4b1e